R/gentlg.R

```diff
@@ -80,6 +80,9 @@
 #' then borders are added to the `colspan` and `colheader` rows. (Default = "old_format").
 #' @param border_fns (optional) List. A list of functions that transform the matrix
 #' passed to `bottom_borders`. Vectorized. See [add_bottom_borders()] for more information.
+#' @param alignments (optional) A list of named lists. Used to specify individual
+#' column or cell alignments. Each named list contains `row`, `col`, and `value`,
+#' which are passed to [huxtable::set_align()] to set the alignments.
```

Comment:
Please, add the mention about the default to make this entry consistent with others.